frontend/templates/troubleshooting/prerendered.tsx

```diff
@@ -72,6 +72,41 @@ const renderStyles: SystemStyleObject = {
       borderColor: 'gray.300',
       padding: '4px',
    },
+
+   '.videoBox_center': {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      clear: 'both',
+   },
+
+   '.imageBox_left': {
+      clear: 'left',
+      float: 'left',
+      marginRight: '30px',
+      '> img': {
+         clear: 'left',
+      },
+   },
+
+   '.imageBox_right': {
+      clear: 'right',
+      float: 'right',
+      marginLeft: '30px',
+      '>img': {
+         clear: 'right',
+      },
+   },
+
+   '.imageBox_center': {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      '>img': {
+         //margin: '0 auto',
```

Comment:
Should this be commented out?

Reply:
It should really be deleted; I'm using flexbox instead of margins for this one.